include/dGramIndex.h

```diff
@@ -27,7 +27,14 @@ class DGramIndex
         std::array<uint8_t, 128> dmap_{}; 
 
 
+    // Rule of 5
     DGramIndex() = default;
+    ~DGramIndex() = default;
+    DGramIndex(const DGramIndex &) = default;
+    DGramIndex & operator=(DGramIndex &) = default;
```

Comment:
There is a const missing here:
```suggestion
    DGramIndex & operator=(const DGramIndex &) = default;
```

Reply:
Sorry, looks like I had an old PR of yours open :/